version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "17.16-SNAPSHOT"
+version in ThisBuild := "17.16"
```

Comment:
I think this is updated during the release anyway, just be sure that `17.17-SNAPSHOT` is pushed back to master when that's finished?